ravedude/src/board.rs

```diff
@@ -70,6 +71,31 @@ impl Board for ArduinoUno {
     }
 }
 
+struct ArduinoMicro;
+
+impl Board for ArduinoMicro {
+    fn display_name(&self) -> &str {
+        "Arduino Micro"
+    }
+
+    fn needs_reset(&self) -> Option<&str> {
+        None
```

Comment:
If this board is similar to Leonardo, you'll need to reset the board manually before programming.  To instruct the user that this is necessary, a message should be added here:

```suggestion
        Some("Reset the board by pressing the reset button once.")
```

Of course you'll need to check whether this is correct - some boards require other reset procedures like e.g. a quick double-press.